sample.env

```diff
@@ -13,4 +13,4 @@ LISTEN_PORT=9000
 REACT_APP_APPBAR_LOGO=iRODS-logo.jpg
 REACT_APP_LOGIN_LOGO=iRODS-logo-1.png
 REACT_APP_BRANDING_NAME=Zone Management Tool
-REACT_APP_PRIMARY_COLOR=#04bdaf
+REACT_APP_PRIMARY_COLOR=#04bdaf
```

Comment:
Let’s get rid of this diff - and then squash again.

Reply:
I'm not really sure why this diff is here. You can view [my code here](https://github.com/ganning127/irods_client_zone_management_tool/blob/109.main.unix-space/sample.env) and the code on irods/main [here](https://github.com/irods/irods_client_zone_management_tool/blob/main/sample.env). They are the same---so not sure why the diff is there

Reply:
i think the newline was removed.

Reply:
aha! that's what it was